components/table/index.css

```diff
@@ -608,48 +482,55 @@
 		border-end-end-radius: var(--mod-table-border-radius, var(--spectrum-table-border-radius));
 	}
 
-	&:focus {
-		outline: 0;
-	}
-
-	&:hover,
+	/* Row focus indicator */
+	&.is-focused,
 	&:focus-visible,
-	&.is-focused {
-		--highcontrast-table-row-text-color: var(--highcontrast-table-row-text-color-hover);
-		--highcontrast-table-icon-color: var(--highcontrast-table-row-text-color-hover);
-		--spectrum-table-cell-background-color: var(--highcontrast-table-row-background-color-hover, var(--mod-table-row-background-color-hover, var(--spectrum-table-row-background-color-hover)));
-	}
+	&:focus {
+		outline: none;
 
-	&:active {
-		--highcontrast-table-row-text-color: var(--highcontrast-table-row-text-color-hover);
-		--highcontrast-table-icon-color: var(--highcontrast-table-row-text-color-hover);
-		--spectrum-table-cell-background-color: var(--highcontrast-table-row-background-color-hover, var(--mod-table-row-active-color, var(--spectrum-table-row-active-color)));
-	}
+		/* First cell of focused rows need fancy new focus indicator borders & no border-radius, and the thicker row focus indicator */
+		.spectrum-Table-cell:first-child {
+			position: relative;
+			border-inline-start: 0;
+			border-block: var(--spectrum-table-row-focus-indicator-outline-width) solid var(--highcontrast-table-focus-indicator-color, var(--mod-table-focus-indicator-color, var(--spectrum-table-focus-indicator-color)));
+
```

Comment:
TODO: Because the ::before element technically removes the table's inline border, the first cell needs the entire cell padding to avoid horizontal layout shift when a row is focused. */

`padding-inline-start: var(--spectrum-table-cell-inline-spacing);`

Reply:
https://github.com/adobe/spectrum-css/pull/3799/commits/c7a3a1ece739856b8e86bcb0e1719b55415a8966